### DISCUSSION TITLE
stops detecting log loops on Haliastur indus_Brahminy Kite

### DISCUSSION THREAD

**Discussion by rdowden2** [mcguirepr89/BirdNET-Pi#1213] [Q&A]
Hi All,

Hopefully its OK to post this here?

I have a Pi 400 which I set up with Birdpi a few weeks ago. It did seem to be working fine for a while but now it stops detecting new birdsong and the log loops on the kite like below:
11:54:53---6.0;9.0-('Haliastur indus_Brahminy Kite', 0.015208226)
11:54:53---9.0;12.0-('Haliastur indus_Brahminy Kite', 0.015208226)
11:54:53---12.0;15.0-('Haliastur indus_Brahminy Kite', 0.015208226)
11:54:56---Files loaded
11:54:56---RECORDING_LENGTH set to 15

And on and on... Every time it hangs/loops it is on the Kite?
It will restart if I reboot the Pi but now that only runs for a little while. I rebooted it yesterday morning and the screen shot below shows it started failing at around 4/5AM today.

![Screenshot from 2024-06-08 12-07-40](https://github.com/mcguirepr89/BirdNET-Pi/assets/13353205/2a3694b3-bfd6-407c-b5a6-71a09ea5d401)

You can listen to live audio on the web page and its fine.
Below is the print_diagnostics_info output. 
========== caddy status ==========
● caddy.service - Caddy
     Loaded: loaded (/lib/systemd/system/caddy.service; enabled; preset: enabled)
    Drop-In: /etc/systemd/system/caddy.service.d
             └─override.conf
     Active: active (running) since Fri 2024-06-07 08:27:18 BST; 1 day 3h ago
       Docs: https://caddyserver.com/docs/
   Main PID: 872 (caddy)
      Tasks: 10 (limit: 3917)
        CPU: 22.330s
     CGroup: /system.slice/caddy.service
             └─872 /usr/bin/caddy run --environ --config /etc/caddy/Caddyfile

Jun 07 08:27:18 birdpi caddy[872]: {"level":"info","ts":1717745238.6593466,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0x4000589080"}
Jun 07 08:27:18 birdpi caddy[872]: {"level":"warn","ts":1717745238.6600375,"logger":"http.auto_https","msg":"server is listening only on the HTTP port, so no automatic HTTPS will be applied to this server","server_name":"srv0","http_port":80}
Jun 07 08:27:18 birdpi caddy[872]: {"level":"info","ts":1717745238.691969,"logger":"http.log","msg":"server running","name":"srv0","protocols":["h1","h2","h3"]}
Jun 07 08:27:18 birdpi caddy[872]: {"level":"info","ts":1717745238.7311096,"msg":"autosaved config (load with --resume flag)","file":"/var/lib/caddy/.config/caddy/autosave.json"}
Jun 07 08:27:18 birdpi caddy[872]: {"level":"info","ts":1717745238.7316225,"msg":"serving initial configuration"}
Jun 07 08:27:18 birdpi systemd[1]: Started caddy.service - Caddy.
Jun 07 08:27:18 birdpi caddy[872]: {"level":"warn","ts":1717745238.8682446,"logger":"tls","msg":"storage cleaning happened too recently; skipping for now","storage":"FileStorage:/var/lib/caddy/.local/share/caddy","instance":"670ab1c0-4040-4b2c-bad9-f4c8da718f2a","try_again":1717831638.868238,"try_again_in":86399.999998222}
Jun 07 08:27:18 birdpi caddy[872]: {"level":"info","ts":1717745238.8685212,"logger":"tls","msg":"finished cleaning storage units"}
Jun 08 08:27:33 birdpi caddy[872]: {"level":"info","ts":1717831653.549314,"logger":"tls","msg":"cleaning storage unit","storage":"FileStorage:/var/lib/caddy/.local/share/caddy"}
Jun 08 08:27:33 birdpi caddy[872]: {"level":"info","ts":1717831653.5498517,"logger":"tls","msg":"finished cleaning storage units"}

========== birdnet_analysis status ==========
● birdnet_analysis.service - BirdNET Analysis
     Loaded: loaded (/etc/systemd/system/birdnet_analysis.service; enabled; preset: enabled)
     Active: active (running) since Sat 2024-06-08 11:57:10 BST; 8s ago
      Until: Sat 2024-06-08 12:12:10 BST; 14min left
   Main PID: 1369386 (bash)
      Tasks: 2 (limit: 3917)
        CPU: 341ms
     CGroup: /system.slice/birdnet_analysis.service
             ├─1369386 bash /usr/local/bin/birdnet_analysis.sh
             └─1369519 sleep 2

Jun 08 11:57:10 birdpi systemd[1]: Started birdnet_analysis.service - BirdNET Analysis.
Jun 08 11:57:11 birdpi birdnet_analysis.sh[1369386]: Files loaded
Jun 08 11:57:12 birdpi birdnet_analysis.sh[1369386]: RECORDING_LENGTH set to 15

========== birdnet_log status ==========
● birdnet_log.service - BirdNET Analysis Log
     Loaded: loaded (/etc/systemd/system/birdnet_log.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 511 (gotty)
      Tasks: 10 (limit: 3917)
        CPU: 5.549s
     CGroup: /system.slice/birdnet_log.service
             └─511 /usr/local/bin/gotty --address localhost -p 8080 -P log --title-format "BirdNET-Pi Log" birdnet_log.sh

Jun 07 11:44:32 birdpi gotty[511]: 2024/06/07 11:44:32 Connection closed by client: 127.0.0.1:58590, connections: 0/0
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36474 200 GET /log/
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36474 200 GET /log/css/index.css
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36484 200 GET /log/css/xterm_customize.css
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36500 200 GET /log/auth_token.js
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36484 200 GET /log/config.js
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36474 200 GET /log/css/xterm.css
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 127.0.0.1:36474 304 GET /log/js/gotty-bundle.js
Jun 08 11:40:26 birdpi gotty[511]: 2024/06/08 11:40:26 New client connected: 127.0.0.1:36506, connections: 1/0
Jun 08 11:40:54 birdpi gotty[511]: 2024/06/08 11:40:54 Connection closed by client: 127.0.0.1:36506, connections: 0/0

========== birdnet_recording status ==========
● birdnet_recording.service - BirdNET Recording
     Loaded: loaded (/etc/systemd/system/birdnet_recording.service; enabled; preset: enabled)
     Active: active (running) since Sat 2024-06-08 04:13:48 BST; 7h ago
   Main PID: 1004990 (bash)
      Tasks: 3 (limit: 3917)
        CPU: 1min 3.788s
     CGroup: /system.slice/birdnet_recording.service
             ├─1004990 bash /usr/local/bin/birdnet_recording.sh
             └─1005016 arecord -f S16_LE -c2 -r48000 -t wav --max-file-time 15 -D default --use-strftime /home/pi/BirdSongs/%B-%Y/%d-%A/%F-birdnet-%H:%M:%S.wav

Jun 08 04:13:48 birdpi birdnet_recording.sh[1004990]: + '[' -z 15 ']'
Jun 08 04:13:48 birdpi birdnet_recording.sh[1004990]: + '[' '!' -z ']'
Jun 08 04:13:48 birdpi birdnet_recording.sh[1004990]: + pulseaudio --check
Jun 08 04:13:49 birdpi birdnet_recording.sh[1004990]: + pgrep arecord
Jun 08 04:13:49 birdpi birdnet_recording.sh[1004990]: + grep 5050 /dev/fd/63
Jun 08 04:13:49 birdpi birdnet_recording.sh[1005014]: ++ netstat -tulpn
Jun 08 04:13:49 birdpi birdnet_recording.sh[1005015]: tcp        0      0 127.0.0.1:5050          0.0.0.0:*               LISTEN      513/python3
Jun 08 04:13:49 birdpi birdnet_recording.sh[1004990]: + '[' -z default ']'
Jun 08 04:13:49 birdpi birdnet_recording.sh[1004990]: + arecord -f S16_LE -c2 -r48000 -t wav --max-file-time 15 -D default --use-strftime /home/pi/BirdSongs/%B-%Y/%d-%A/%F-birdnet-%H:%M:%S.wav
Jun 08 04:13:49 birdpi birdnet_recording.sh[1005016]: Recording WAVE '/home/pi/BirdSongs/%B-%Y/%d-%A/%F-birdnet-%H:%M:%S.wav' : Signed 16 bit Little Endian, Rate 48000 Hz, Stereo

========== birdnet_server status ==========
● birdnet_server.service - BirdNET Analysis Server
     Loaded: loaded (/etc/systemd/system/birdnet_server.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 513 (python3)
      Tasks: 11 (limit: 3917)
        CPU: 3h 35min 13.534s
     CGroup: /system.slice/birdnet_server.service
             └─513 /home/pi/BirdNET-Pi/birdnet/bin/python3 /usr/local/bin/server.py

Jun 08 11:56:36 birdpi python3[513]: READING AUDIO DATA... DONE! READ 5 CHUNKS.
Jun 08 11:56:36 birdpi python3[513]: Date-time: 2024-06-08 11:56:20
Jun 08 11:56:51 birdpi python3[513]: ANALYZING AUDIO... DONE! Time 1.4 SECONDS
Jun 08 11:56:51 birdpi python3[513]: WRITING RESULTS TO /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:56:20.wav.csv ... DONE! WROTE 0 RESULTS.
Jun 08 11:56:51 birdpi python3[513]: READING AUDIO DATA... DONE! READ 5 CHUNKS.
Jun 08 11:56:51 birdpi python3[513]: Date-time: 2024-06-08 11:56:35
Jun 08 11:57:07 birdpi python3[513]: ANALYZING AUDIO... DONE! Time 1.4 SECONDS
Jun 08 11:57:07 birdpi python3[513]: WRITING RESULTS TO /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:56:35.wav.csv ... DONE! WROTE 0 RESULTS.
Jun 08 11:57:07 birdpi python3[513]: READING AUDIO DATA... DONE! READ 5 CHUNKS.
Jun 08 11:57:07 birdpi python3[513]: Date-time: 2024-06-08 11:56:50

========== birdnet_stats status ==========
● birdnet_stats.service - BirdNET Stats
     Loaded: loaded (/etc/systemd/system/birdnet_stats.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 520 (streamlit)
      Tasks: 8 (limit: 3917)
        CPU: 1min 10.739s
     CGroup: /system.slice/birdnet_stats.service
             └─520 /home/pi/BirdNET-Pi/birdnet/bin/python3 /home/pi/BirdNET-Pi/birdnet/bin/streamlit run /home/pi/BirdNET-Pi/scripts/plotly_streamlit.py --browser.gatherUsageStats false --server.address localhost --server.baseUrlPath /stats

Jun 07 08:27:08 birdpi systemd[1]: Started birdnet_stats.service - BirdNET Stats.
Jun 07 08:27:23 birdpi streamlit[520]:   You can now view your Streamlit app in your browser.
Jun 07 08:27:23 birdpi streamlit[520]:   URL: http://localhost:8501/stats

========== chart_viewer status ==========
● chart_viewer.service - BirdNET-Pi Chart Viewer Service
     Loaded: loaded (/etc/systemd/system/chart_viewer.service; enabled; preset: enabled)
     Active: activating (auto-restart) since Sat 2024-06-08 11:55:43 BST; 1min 36s ago
    Process: 1368108 ExecStart=/home/pi/BirdNET-Pi/birdnet/bin/python3 /usr/local/bin/daily_plot.py (code=exited, status=0/SUCCESS)
   Main PID: 1368108 (code=exited, status=0/SUCCESS)
        CPU: 9.093s

Jun 08 11:55:43 birdpi systemd[1]: chart_viewer.service: Deactivated successfully.
Jun 08 11:55:43 birdpi systemd[1]: chart_viewer.service: Consumed 9.093s CPU time.

========== extraction status ==========
● extraction.service - BirdNET BirdSound Extraction
     Loaded: loaded (/etc/systemd/system/extraction.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 536 (bash)
      Tasks: 2 (limit: 3917)
        CPU: 2h 34min 8.891s
     CGroup: /system.slice/extraction.service
             ├─    536 bash -c "while true;do extract_new_birdsounds.sh;sleep 3;done"
             └─1369516 sleep 3

Jun 08 11:54:56 birdpi env[1367611]: Line = 2024-06-08-birdnet-11:54:35.wav.csv
Jun 08 11:55:13 birdpi env[1367827]: Line = 2024-06-08-birdnet-11:54:50.wav.csv
Jun 08 11:55:29 birdpi env[1368048]: Line = 2024-06-08-birdnet-11:55:05.wav.csv
Jun 08 11:55:42 birdpi env[1368220]: Line = 2024-06-08-birdnet-11:55:20.wav.csv
Jun 08 11:55:56 birdpi env[1368367]: Line = 2024-06-08-birdnet-11:55:35.wav.csv
Jun 08 11:56:12 birdpi env[1368608]: Line = 2024-06-08-birdnet-11:55:50.wav.csv
Jun 08 11:56:28 birdpi env[1368810]: Line = 2024-06-08-birdnet-11:56:05.wav.csv
Jun 08 11:56:42 birdpi env[1368983]: Line = 2024-06-08-birdnet-11:56:20.wav.csv
Jun 08 11:56:58 birdpi env[1369189]: Line = 2024-06-08-birdnet-11:56:35.wav.csv
Jun 08 11:57:14 birdpi env[1369460]: Line = 2024-06-08-birdnet-11:56:50.wav.csv

========== web_terminal status ==========
● web_terminal.service - BirdNET-Pi Web Terminal
     Loaded: loaded (/etc/systemd/system/web_terminal.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 561 (gotty)
      Tasks: 7 (limit: 3917)
        CPU: 122ms
     CGroup: /system.slice/web_terminal.service
             └─561 /usr/local/bin/gotty --address localhost -w -p 8888 -P terminal --title-format "BirdNET-Pi Terminal" login

Jun 07 08:27:08 birdpi systemd[1]: Started web_terminal.service - BirdNET-Pi Web Terminal.
Jun 07 08:27:10 birdpi gotty[561]: 2024/06/07 08:27:09 GoTTY is starting with command: login
Jun 07 08:27:10 birdpi gotty[561]: 2024/06/07 08:27:09 Permitting clients to write input to the PTY.
Jun 07 08:27:10 birdpi gotty[561]: 2024/06/07 08:27:09 HTTP server is listening at: http://127.0.0.1:8888/terminal/

========== spectrogram_viewer status ==========
● spectrogram_viewer.service - BirdNET-Pi Spectrogram Viewer
     Loaded: loaded (/etc/systemd/system/spectrogram_viewer.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:08 BST; 1 day 3h ago
   Main PID: 553 (bash)
      Tasks: 2 (limit: 3917)
        CPU: 59min 54.925s
     CGroup: /system.slice/spectrogram_viewer.service
             ├─    553 bash /usr/local/bin/spectrogram.sh
             └─1369495 sleep 3

Jun 08 11:57:13 birdpi spectrogram.sh[553]: + sox -V1 /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav -n remix 1 rate 24k spectrogram -c BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav -o /home/pi/BirdSongs/Extracted/spectrogram.png
Jun 08 11:57:13 birdpi spectrogram.sh[553]: + sleep 3
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + true
Jun 08 11:57:16 birdpi spectrogram.sh[1369493]: ++ cat /home/pi/BirdNET-Pi/analyzing_now.txt
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + analyzing_now=/home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + '[' '!' -z /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav ']'
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + '[' -f /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav ']'
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + spectrogram_png=/home/pi/BirdSongs/Extracted/spectrogram.png
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + sox -V1 /home/pi/BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav -n remix 1 rate 24k spectrogram -c BirdSongs/June-2024/08-Saturday/2024-06-08-birdnet-11:57:05.wav -o /home/pi/BirdSongs/Extracted/spectrogram.png
Jun 08 11:57:16 birdpi spectrogram.sh[553]: + sleep 3

========== livestream status ==========
● livestream.service - BirdNET-Pi Live Stream
     Loaded: loaded (/etc/systemd/system/livestream.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-06-07 08:27:15 BST; 1 day 3h ago
   Main PID: 879 (bash)
      Tasks: 6 (limit: 3917)
        CPU: 1h 32min 4.615s
     CGroup: /system.slice/livestream.service
             ├─879 bash /usr/local/bin/livestream.sh
             └─883 ffmpeg -nostdin -loglevel error -ac 2 -f alsa -i default -acodec libmp3lame -b:a 320k -ac 2 -content_type audio/mpeg -f mp3 icecast://source:birdnetpi@localhost:8000/stream -re

Jun 08 11:56:59 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751041563 >= 4751041523
Jun 08 11:57:02 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751174239 >= 4751174167
Jun 08 11:57:03 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751208771 >= 4751208742
Jun 08 11:57:03 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751238555 >= 4751238526
Jun 08 11:57:05 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751325466 >= 4751325316
Jun 08 11:57:05 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751325466 >= 4751324871
Jun 08 11:57:09 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751509124 >= 4751508886
Jun 08 11:57:14 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751727027 >= 4751727001
Jun 08 11:57:15 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751809387 >= 4751809386
Jun 08 11:57:17 birdpi livestream.sh[883]: [mp3 @ 0x55824a4320] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 4751880515 >= 4751880415

========= Syslog snippet ==========
========== Mounted File Systems ==========
Filesystem      Size  Used Avail Use% Mounted on
udev            1.6G     0  1.6G   0% /dev
tmpfs           380M  1.3M  378M   1% /run
/dev/mmcblk0p2   29G   20G  7.4G  74% /
tmpfs           1.9G     0  1.9G   0% /dev/shm
tmpfs           5.0M   16K  5.0M   1% /run/lock
/dev/mmcblk0p1  510M   63M  448M  13% /boot/firmware
tmpfs           380M  4.0K  380M   1% /run/user/1000

========== Memory Usage ==========
               total        used        free      shared  buff/cache   available
Mem:           3.7Gi       596Mi       1.4Gi       4.1Mi       1.8Gi       3.1Gi
Swap:           99Mi          0B        99Mi

========== Load Averages ==========
 11:57:19 up 1 day,  3:30,  2 users,  load average: 0.33, 0.44, 0.55

========== CPU Info ==========
processor	: 0
BogoMIPS	: 108.00
Features	: fp asimd evtstrm crc32 cpuid
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x0
CPU part	: 0xd08
CPU revision	: 3

processor	: 1
BogoMIPS	: 108.00
Features	: fp asimd evtstrm crc32 cpuid
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x0
CPU part	: 0xd08
CPU revision	: 3

processor	: 2
BogoMIPS	: 108.00
Features	: fp asimd evtstrm crc32 cpuid
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x0
CPU part	: 0xd08
CPU revision	: 3

processor	: 3
BogoMIPS	: 108.00
Features	: fp asimd evtstrm crc32 cpuid
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x0
CPU part	: 0xd08
CPU revision	: 3

Revision	: c03130
Serial		: 100000008ebb71cf
Model		: Raspberry Pi 400 Rev 1.0

========== System Temperature ==========
CPU Temperature: 31.64°C / 88.95°F

========== Extra Info ==========
........................................IPs.....................................
########
########
..................................`vcgencmd stats`..............................
throttled=0x0
Binary: 0
....................................Clock Speeds................................
arm:	frequency(48)=1800457088
core:	frequency(1)=500000992
h264:	frequency(28)=0
isp:	frequency(45)=0
v3d:	frequency(46)=500000992
uart:	frequency(22)=48001464
pwm:	frequency(25)=0
emmc:	frequency(50)=250000496
pixel:	frequency(29)=74988280
vec:	frequency(10)=0
hdmi:	frequency(9)=119997072
dpi:	frequency(4)=0
........................................Volts...................................
core:	volt=0.9160V
sdram_c:	volt=1.1000V
sdram_i:	volt=1.1000V
sdram_p:	volt=1.1000V
.....................................Caddyfile..................................
http:// {
	root * /home/pi/BirdSongs/Extracted
	file_server browse
	handle /By_Date/* {
		file_server browse
	}
	handle /Charts/* {
		file_server browse
	}
	basicauth /views.php?view=File* {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	basicauth /Processed* {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	basicauth /scripts* {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	basicauth /stream {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	basicauth /phpsysinfo* {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	basicauth /terminal* {
		birdnet $2a$14$WcEZVheYiw12HCweFgYKF.gD9PLYL0poNy03qWmisCUILMgS6ywPK
	}
	reverse_proxy /stream localhost:8000
	php_fastcgi unix//run/php/php8.2-fpm.sock
	reverse_proxy /log* localhost:8080
	reverse_proxy /stats* localhost:8501
	reverse_proxy /terminal* localhost:8888
}
.................................... Crontab....................................

SHELL=/bin/sh
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin

17 *	* * *	root	cd / && run-parts --report /etc/cron.hourly
25 6	* * *	root	test -x /usr/sbin/anacron || { cd / && run-parts --report /etc/cron.daily; }
47 6	* * 7	root	test -x /usr/sbin/anacron || { cd / && run-parts --report /etc/cron.weekly; }
52 6	1 * *	root	test -x /usr/sbin/anacron || { cd / && run-parts --report /etc/cron.monthly; }
*/5 * * * * pi /usr/local/bin/disk_check.sh >/dev/null 2>&1
*/3 * * * * pi /usr/local/bin/cleanup.sh >/dev/null 2>&1
@reboot pi /usr/local/bin/cleanup.sh >/dev/null 2>&1
0 1 * * 1 pi /usr/local/bin/weekly_report.sh >/dev/null 2>&1
*/5 * * * * pi /usr/local/bin/disk_check.sh >/dev/null 2>&1
*/3 * * * * pi /usr/local/bin/cleanup.sh >/dev/null 2>&1
@reboot pi /usr/local/bin/cleanup.sh >/dev/null 2>&1
0 1 * * 1 pi /usr/local/bin/weekly_report.sh >/dev/null 2>&1

========== Connected Microphone Devices ==========
**** List of CAPTURE Hardware Devices ****
card 0: Microphone [USB Microphone], device 0: USB Audio [USB Audio]
  Subdevices: 0/1
  Subdevice #0: subdevice #0

null
    Discard all samples (playback) or generate zero samples (capture)
default
    Playback/recording through the PulseAudio sound server
sysdefault
    Default Audio Device
lavrate
    Rate Converter Plugin Using Libav/FFmpeg Library
samplerate
    Rate Converter Plugin Using Samplerate Library
speexrate
    Rate Converter Plugin Using Speex Resampler
jack
    JACK Audio Connection Kit
oss
    Open Sound System
pulse
    PulseAudio Sound Server
speex
    Plugin using Speex DSP (resample, agc, denoise, echo, dereverb)
upmix
    Plugin for channel upmix (4,6,8)
vdownmix
    Plugin for channel downmix (stereo) with a simple spacialization
hw:CARD=Microphone,DEV=0
    USB Microphone, USB Audio
    Direct hardware device without any conversions
plughw:CARD=Microphone,DEV=0
    USB Microphone, USB Audio
    Hardware device with all software conversions
sysdefault:CARD=Microphone
    USB Microphone, USB Audio
    Default Audio Device
front:CARD=Microphone,DEV=0
    USB Microphone, USB Audio
    Front output / input
dsnoop:CARD=Microphone,DEV=0
    USB Microphone, USB Audio
    Direct sample snooping device
usbstream:CARD=Microphone
    USB Microphone
    USB Stream Output
usbstream:CARD=vc4hdmi0
    vc4-hdmi-0
    USB Stream Output
usbstream:CARD=vc4hdmi1
    vc4-hdmi-1
    USB Stream Output

========= Date and Time ==========
Sat  8 Jun 11:57:19 BST 2024
===========================================

The only errors I can see are those about the stream
"application provided invalid, non monotonically increasing dts to muxer in stream"

It always seems to stick on the Kite detection which made me think it was the model so I switched it and it still does the same. 
Can anyone offer some advice on where to look next?

Cheers

Roger

**Comment by Scraggarax** [comment]
Heads up, I read the first and last bits, and skimmed the rest. That's a lot of stuff.

This seems like what I reported in issue #653 Give that thread a read and see if that helps at all.

**Reply by rdowden2** [reply]
Hi,

I had seen someone else post the "diagnostic" script output so I ran it too which produces the large output. I do have plenty of storage space at the moment. I will take a look and see if there is a big back log but it seems to be more a loop rather than too much to catch up with type issue. I did reboot today and it started processing and listing detection's after about 20 mins. I'll see how long it lasts this time. 

Thanks for the feedback

Roger

**Comment by rdowden2** [comment]
Hi,

Quick update, it lasted till the end of the day and then back into the looping. I tried restarting the services one by one but it made no difference. I checked it today, still no detection's. You can see from the log, it is processing the latest logs but just detecting the kite.
08:58:11---Date-time: 2024-06-12 08:57:56
08:58:12---0.0;3.0-('Haliastur indus_Brahminy Kite', 0.015208226)
08:58:12---3.0;6.0-('Haliastur indus_Brahminy Kite', 0.015208226)
08:58:12---6.0;9.0-('Haliastur indus_Brahminy Kite', 0.015208226)
08:58:12---9.0;12.0-('Haliastur indus_Brahminy Kite', 0.015208226)
08:58:12---12.0;15.0-('Haliastur indus_Brahminy Kite', 0.015208226)
08:58:15---Files loaded
08:58:16---RECORDING_LENGTH set to 15
**08:58:26---BirdNET-Pi/birdnet/bin/python3 BirdNET-Pi/scripts/analyze.py --i BirdSongs/June-2024/12-Wednesday/2024-06-12-birdnet-08:58:11.wav --o BirdSongs/June-2024/12-Wednesday/2024-06-12-birdnet-08:58:11.wav.csv --lat 51.XX --lon 0.XX --week 22 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7**
08:58:26---ANALYZING AUDIO... DONE! Time 1.4 SECONDS
08:58:26---WRITING RESULTS TO BirdSongs/June-2024/12-Wednesday/2024-06-12-birdnet-08:57:56.wav.csv ... DONE! WROTE 0 RESULTS.

Cheers

Roger

**Reply by rdowden2** [reply]
Hi,

I did an update via the tools system controls and it has now been running successfully for a day or so. I will stop the daily reboots and see how it continues.

Cheers

Roger

**Comment by rdowden2** [comment]
Hi All,

Nope the update did not fix it. Looks like I need to reinstall.

Roger
